src/helics/core/ProfilerBuffer.cpp

```diff
@@ -54,4 +54,24 @@ void ProfilerBuffer::writeFile()
     mBuffers.clear();
 }
 
+void ProfilerBuffer::setOutputFile(std::string fileName)
+{
+    if (fileName.empty()) {
+        mFileName.clear();
+        return;
+    }
+    if (fileName.front() == '+') {
+        mFileName = fileName.substr(1);
+        return;
```

Comment:
I think this would be better as a separate option that changes log files to append mode -- file and folder names with a `+` are valid on Windows and Linux.

Reply:
I modified it so there is a separate --profiler_append=filename option instead of the '+'    I didn't like the idea of a separate option that would modify the --profiler,  I though that would be confusing